src/vterm.hpp

```diff
@@ -87,10 +87,11 @@ class VTermToFBInk {
         // only call this from main (yeah bad code...)
         // because we need to resize the pty too
         int res = fbink_reinit(fbfd, &config);
-        if ((res == OK_BPP_CHANGE) || (res == OK_ROTA_CHANGE)) {
-            /* if both changed, OK_BPP_CHANGE `wins' */
+        if (res > EXIT_SUCCESS && res & OK_LAYOUT_CHANGE) {
+            /* we only actually care about layout changes */
             fbink_get_state(&config, &state);
             printf("fbink_reinit()\n");
+            vterm_screen_reset(screen, 1);
```

Comment:
Nice. Didn't know about `vterm_screen_reset()`

Reply:
Yeah, I went grep hunting, and it appears to do everything I wanted it do: a screen clear, a terminal clear, resetting positions, the whole shebang ;).

Reply:
Order seems relevant, too, because doing it *after* the resize did not help prevent my crash repro ^^.